packages/angular/src/app/components/utils/agenda/agenda.component.ts

```diff
@@ -11,10 +11,12 @@ export type AgendaItem = { startDate: Date };
 @Component({
   selector: 'agenda',
   template: `
-  <dx-list [dataSource]="items"
-           (onItemClick)="handleItemClick($event)">
-    <div *dxTemplate="let task of 'item'"
-         class="agenda-item"
+  <dx-list
+    [dataSource]="items"
+    (onItemClick)="handleItemClick($event)">
```

Comment:
being totally obsessive: a new line before `>` is missing

Reply:
Hmm.... it strange. If will check prettier settings